handlers/group.ts

```diff
@@ -0,0 +1,62 @@
+import {randomUUID} from 'expo-crypto';
+interface IGroup {
+    create(groupName: string, action : () => void): void;
+    join(uuid: string, action : (ack: Boolean) => void): void;
+    checkIfInGroup(action : (ack: Boolean) => void): void ;
+    leave() : Boolean;
+    setPeers() : void;
+}
+
+class GroupHandle implements IGroup {
+
+    constructor(private gun: Gun) {        
+    }
+
+    public create(groupName: string, action : () => void): void {
+        let user = gun.user(userPub)
+        let groupId : string = randomUUID();        
+        let context = gun.get("groups").set({groupId: groupId})
+        context.get("members").set({members: userPub})
```

Comment:
Isn't the proper use `context.get("members").set(userPub)` as this will add userPub to the set members?

Reply:
or maybe `set({member: userPub})`

Reply:
DB structure should  be as we want now, grouping just works